index.html

```diff
@@ -438,6 +438,16 @@ <h2>
         />
       </a>
     </footer>
+    <div class="menu-bar">
+      <i class="fa fa-times"></i>
+      <div class="header-slide-links">
+        <li><a >Portifolio</a></li>
+        <li><a >About</a></li>
+        <!-- <li><a >Contact</a></li> -->
+        <li><a href="#contact">Contact</a></li>
+      </div>
+    </div>
```

Comment:
- [ ] I would urge you to please aspire to make your design look as close as possible to the figma design. 

your design:
![image](https://user-images.githubusercontent.com/107795045/210777712-696572c6-7b68-4282-a5c7-c4937c3a9e00.png)

template:
![image](https://user-images.githubusercontent.com/107795045/210777878-b3571996-69b0-46c2-908c-608f36c62a08.png)

![image](https://user-images.githubusercontent.com/107795045/210778339-6651faa8-a385-4a2e-aed0-c56b686fa82c.png)